### PR TITLE
Null rod radial menu update

### PR DESCRIPTION
**Pull request by Arkatos1**
## About The Pull Request

This PR does following things:
- Updates radial menu for a null rod reskin variant selection. It will now also include a small button for showing additional information about each given null rod type. No need to wiki dive to find out what each type does.
- Cleans up relevant code a little bit and provides proper documentation.

**Example image:**

NullrodInfo

## Why It's Good For The Game

Less reliance on the wiki, quality of life for chaplains.

## Changelog
:cl: Arkatos
qol: Updated radial menu for a null rod reskin variant selection. It will now include a small button for showing additional information about each given null rod variant.
/:cl:

### CODE REVIEW

File: code/game/objects/items/holy_weapons.dm [hunk 1]
```diff
@@ -291,35 +298,38 @@
...
-	name = "god hand"
-	desc = "This hand of yours glows with an awesome power!"
 	slot_flags = null
 	item_flags = ABSTRACT | DROPDEL
 	w_class = WEIGHT_CLASS_HUGE
 	hitsound = 'sound/weapons/sear.ogg'
 	damtype = BURN
 	attack_verb_continuous = list("punches", "cross counters", "pummels")
 	attack_verb_simple = list("punch", "cross counter", "pummel")
+	menu_description = "An undroppable god hand dealing a burn damage. Disappears if the arm holding it is cut off."
```
Comment (Fikou) [inline code/game/objects/items/holy_weapons.dm:314]:
```suggestion
	menu_description = "An undroppable god hand dealing burn damage. Disappears if the arm holding it is cut off."
```

File: code/game/objects/items/holy_weapons.dm [hunk 2]
```diff
@@ -446,64 +464,70 @@
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	attack_verb_continuous = list("attacks", "slashes", "stabs", "slices", "tears", "lacerates", "rips", "dices", "cuts")
 	attack_verb_simple = list("attack", "slash", "stab", "slice", "tear", "lacerate", "rip", "dice", "cut")
+	menu_description = "An odd s(w)ord dealing a laughable amount of damage. Fits in pockets. Can be worn on the belt."
```
Comment (Fikou) [inline code/game/objects/items/holy_weapons.dm:467]: doesnt say its the strongest weapon 0/10

File: code/game/objects/items/holy_weapons.dm [hunk 3]
```diff
@@ -572,6 +597,7 @@
 	w_class = WEIGHT_CLASS_BULKY
 	attack_verb_continuous = list("smashes", "bashes", "hammers", "crunches")
 	attack_verb_simple = list("smash", "bash", "hammer", "crunch")
+	menu_description = "A war hammer. Capable of tapping knees to measure brain health. Can be worn on the belt."
```
Comment (Fikou) [inline code/game/objects/items/holy_weapons.dm:600]: it can also be used for kneecapping

Comment (Arkatos1) [reply code/game/objects/items/holy_weapons.dm:600]: #59816 added kneecapping only for the baseball bats so far, so I would not be so sure about that. The only element that is added in Initialize() is just kneejerking.

**Comment by MrDoomBringer** [comment]
This has been needed for so damn long, good work

File: code/game/objects/items/holy_weapons.dm [hunk 4]
```diff
@@ -733,6 +768,7 @@
 	worn_icon_state = "bostaff0"
 	lefthand_file = 'icons/mob/inhands/weapons/staves_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/weapons/staves_righthand.dmi'
+	menu_description = "A staff which provides a medium chance of blocking incoming melee attacks and deals a little less damage due to it's wood material. Can be worn on the back."
```
Comment (MrDoomBringer) [inline code/game/objects/items/holy_weapons.dm:771]:
```suggestion
	menu_description = "A staff which provides a medium chance of blocking incoming melee attacks and deals a little less damage due to being made of wood. Can be worn on the back."
```

**Comment by Mothblocks** [comment]
i was tihnking about the radial info buttons the other day only now learned the only other thing theyre on (other than blobs) is spiders

**Pull request merged**